.github/workflows/test.yml

```
@@ -13,7 +13,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
```

nit: I recommend add options for `fail-fast: false` so that all tests will run.

```suggestion
      fail-fast: false
      matrix:
```